Makefile.am

```diff
@@ -64,6 +66,7 @@ endif
 src_libfabric_la_CPPFLAGS = $(AM_CPPFLAGS)
 src_libfabric_la_LDFLAGS =
 src_libfabric_la_LIBADD =
+src_libfabric_la_DEPENDENCIES = $(srcdir)/libfabric.map
```

Comment:
Should this be part of a different commit or PR?  Doesn't seem related to PSM/PSM2.